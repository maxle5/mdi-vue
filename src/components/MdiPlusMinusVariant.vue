
  <template>
      <svg :width="size" :height="size" :viewBox="viewBox">
          <path d="M3 7H6V4H8V7H11V9H8V12H6V9H3V7M13 15H21V17H13V15M16.04 3H18.35L7.96 21H5.65L16.04 3Z" style="fill: currentColor" />
      </svg>
  </template>

  <script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiPlusMinusVariant",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
      computed: {
        viewBox() {
          return `0 0 ${this.size} ${this.size}`;
        }
      }
  })
  </script>
