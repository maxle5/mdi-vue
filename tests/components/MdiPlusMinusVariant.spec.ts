
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiPlusMinusVariant from "../../src/components/MdiPlusMinusVariant.vue";

test("MdiPlusMinusVariant snapshot", () => {
  const wrapper = mount(MdiPlusMinusVariant, {});
  expect(wrapper.html()).toMatchSnapshot();
});
