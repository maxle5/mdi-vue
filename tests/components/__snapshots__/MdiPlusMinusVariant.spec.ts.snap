// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiPlusMinusVariant snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M3 7H6V4H8V7H11V9H8V12H6V9H3V7M13 15H21V17H13V15M16.04 3H18.35L7.96 21H5.65L16.04 3Z" style="fill: currentColor;"></path>
</svg>"
`;
